### PR TITLE
Fixed tests on Node 9

### PR DESCRIPTION
**Pull request by mcollina**
Fixes https://github.com/pinojs/pino/issues/339

### CODE REVIEW

File: test/events.test.js [hunk 1]
```diff
@@ -97,7 +98,11 @@ test('terminates on SIGHUP when no other handlers registered', function (t) {
   }))
 
   child.on('exit', function (code) {
-    t.is(code, 0)
+    if (semver.gt(process.version, '9.0.0')) {
+      t.is(code, 1)
```
Comment (jsumners) [inline test/events.test.js]: Why would the return code suddenly be `1` on Node >=9.0.0?

Comment (mcollina) [reply test/events.test.js]: I have no idea. However I have reproduced it even without pino, if you send SIGHUP on Node 9, you get an exit code of 1.

Comment (jsumners) [reply test/events.test.js]: There's something off here. When I run the following script with 9.3.0 and send it `kill -HUP`, I get an exit code of `0`:

```js
'use strict'

process.on('SIGHUP', () => {
  console.log(arguments)
  process.exit()
})

setInterval(() => {}, 100)
```

output:
```js
{ '0': {},
  '1':
   { [Function: require]
     resolve: { [Function: resolve] paths: [Function: paths] },
     main:
      Module {
        id: '.',
        exports: {},
        parent: null,
        filename: '/private/tmp/19/foo.js',
        loaded: true,
        children: [],
        paths: [Array] },
     extensions: { '.js': [Function], '.json': [Function], '.node': [Function] },
     cache: { '/private/tmp/19/foo.js': [Module] } },
  '2':
   Module {
     id: '.',
     exports: {},
     parent: null,
     filename: '/private/tmp/19/foo.js',
     loaded: true,
     children: [],
     paths:
      [ '/private/tmp/19/node_modules',
        '/private/tmp/node_modules',
        '/private/node_modules',
        '/node_modules' ] },
  '3': '/private/tmp/19/foo.js',
  '4': '/private/tmp/19' }
```

**Comment by mcollina** [comment]
Closed in favour of https://github.com/pinojs/pino/pull/342.

**Pull request closed without merging**